### PR TITLE
Add number boundary validation to Transaction.value

### PR DESCRIPTION
**Pull request by lozette**
## Changes in this PR

Trello: https://trello.com/c/cKUqF0VW/141-add-transaction-value-validation

The value of Transaction.value must be between 1 and 99,999,999,999 (or, 100
billion minus one)

### CODE REVIEW

File: spec/factories/transaction.rb [hunk 1]
```diff
@@ -9,5 +9,6 @@
     value { 110.01 }
     disbursement_channel { "1" }
     currency { "gbp" }
+    association :fund, factory: :fund
```
[the remark this answers is not in the document]
Comment (lozette) [reply spec/factories/transaction.rb]: No, I will pull it out and put it into my work for https://trello.com/c/RXlGPpJQ/126-refactor-create-fund-and-user-factories-with-associations as it belongs there!

**Comment by tahb** [comment]
@lozette Back with you for a review and testing of my additonal comments, any questions let me know 👍

**Comment by lozette** [comment]
I've rebased this and reviewed, am v happy with it 👍  but I think we need a third party to review it now!

**Comment by tahb** [comment]
@lozette Thanks for looking at this again. It would be nice to have another pair of eyes but realistically I'm not sure we know when some will be free to help. I'm happy that we've both been quite involved with this and given it 2 pairs of eyes. As normal If we've missed something we can come back and iterate it, while we have no users it's low risk.

**Comment by lozette** [comment]
OK! I am unable to approve it so if you can, go for it!

**Pull request merged**